src/node_crypto_common.cc

```diff
@@ -386,31 +386,27 @@ Local<Value> ToV8Value(Environment* env, const BIOPointer& bio) {
   return ret.FromMaybe(Local<Value>());
 }
 
-MaybeLocal<Value> GetCipherName(
-    Environment* env,
-    const SSL_CIPHER* cipher) {
+MaybeLocal<Value> GetCipherValue(Environment* env,
+    const SSL_CIPHER* cipher,
+    std::function<const char*(const SSL_CIPHER* cipher)> getstr) {
```

Comment:
```suggestion
    const char* (*getstr)(const SSL_CIPHER* cipher)) {
```
`std::function` is not necessary here and adds a bit of overhead; the typical implementation is at least two or three pointers big.

Reply:
Ah good point, I'll update this shortly. Thanks!